### PR TITLE
Update hung dg

### PR DESCRIPTION
**Pull request by nhamhung**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -375,67 +375,60 @@ be switch-cased again in `ABCCommand`.
...
-Below is an example usage scenario and how the edit user data command works.
+Step 1. User launches the ResuMe application for the first time. The user profile data is not yet edited and will thus be
+initialized with the initial json data stored.
 
-* User launches the application for the first time. The User Profile and Data will be initialized with the initial json
-data stored.
+image::user-default.png[]
 
-image::user-default-data.png[]
+Step 2. User executes `me dp/FILEPATH n/NAME d/DESCRIPTION p/PHONE e/EMAIL g/GITHUB u/UNIVERSITY m/MAJOR f/FROM t/TO c/CAP`
```
Comment (duongphammmm) [inline docs/DeveloperGuide.adoc]: try to leave a white space between the prefix and the value to make it consistent with the rest of the document, eg. me dp/ FILE_PATH

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -453,23 +446,112 @@ This design is chosen because `EditUserCommand` only concerns with a single `Per
...
+
+** Pros: Make ResuMe more inline with being a resume building application.
+** Cons: May miss out on a portion of users who would appreciate this feature, especially those with a habit of jotting down notes.
+
+*Aspect: Whether `Note` class should extend `Item` class*
+
+* *Alternative 1 (current choice):* `Note` is also an `Item`
+
+This design is chosen because note taking feature is intended to have similar `Command` to a typical `Item` such as `AddCommand`, `EditCommand` and `SortCommand`.
+As such, by extending `Item`, `Note` can inherit attributes such as `Index` and `TAG` as well as being able to kept as a
```
Comment (duongphammmm) [inline docs/DeveloperGuide.adoc]:
```suggestion
As such, by extending `Item`, `Note` can inherit attributes such as `Index` and `Tag` as well as being able to kept as a
```

**Review by duongphammmm — APPROVED** [review]
LGTM! Could consider fixing some format consistency issue like suggested before merging 👍

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (nhamhung) [reply docs/DeveloperGuide.adoc]: Thank you sis :)

**Pull request merged**